src/utilities.cpp

```diff
@@ -85,8 +85,9 @@ std::vector<int> aste::setupMesh(precice::SolverInterface &interface, const aste
   auto tconnectivity = std::chrono::steady_clock::now();
   VLOG(1) << "Mesh Setup: 2) Edges";
   const auto edgeMap = setupEdgeIDs(interface, mesh, meshID, vertexIDs);
-
-  VLOG(1) << "Mesh Setup: 3) Triangles";
+  if (!mesh.triangles.empty()) {
+    VLOG(1) << "Mesh Setup: 3) Triangles";
+  }
```

Comment:
We could either add an `else` branch with `skipped Triangles` or directly state the number of entities in the log, e.g. `Setup <N> triangles for mesh <Name>`

Reply:
Entity quantities are added